Automattic/jetpack
# My Jetpack: allow dismissing the IDC modal

Related to #32249

## Proposed changes:
* Allow dismissing the IDC modal in My Jetpack section.

The new IDC modal blocks the entire screen, so admins are forced to make a decision and cannot leave the page, even the sidebar menu is blocked.
Since "My Jetpack" is now the default Jetpack section, the problem will be even worse.

This PR will make the IDC modal dismissible by clicking anywhere outside the modal or pressing Esc.

### Other information:


## Jetpack product discussion
p1689693278838909-slack-C02NENKA4GN

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Connect Jetpack, install the [Jetpack Debug Helper plugin](https://github.com/Automattic/jetpack-debug-helper/archive/refs/heads/trunk.zip).
2. Go to "Jetpack Debug -> IDC Simulator", enter the "Spoof URL" and enable "IDC Simulation". Then click on "Send Remote Request" to trigger IDC.
3. Go to "Jetpack -> My Jetpack", confirm that the modal shows up.
4. Click anywhere on the page other than the modal itself, confirm the modal disappears.
5. Reload the page, you should see the modal again.
